draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -781,6 +781,22 @@ Verifiers need to ensure that they are not using expired or revoked keys
 for signature verification using reasonable and appropriate means for the given
 key-distribution method.
 
+## Selective Disclosure without Holder Binding
```

Comment:
This is a somewhat nondescript section title and feels duplicate to the "Holder Binding" title above; how about this?
```suggestion
## Forwarding Credentials
```

Reply:
@Sakurann before I accept, what do you think?

Reply:
I think this is relevant to #254. Is `Credentials` a term used for SD-JWT + disclosures..?